apiary.apib

```diff
@@ -523,8 +565,8 @@ It will be called with two arguments:
 
 # Customize the Session Interface
 
-After the iframe has been created, it can be controlled via postMessage and it
-will emit messages back to the parent frame. One can start a Surfly session with
+After the iframe has been created, it can be controlled via postMessage and
+will emit messages back to the parent frame. You can start a Surfly session with
 the UI=OFF flag as described above.
```

Comment:
`UI=OFF` is probably something old. We use `ui_off` instead. See line 236. I don't know it it is worth to mentioning it here. Should it be just a link to the section where we describe all options?